Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -157,14 +157,13 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
     ASSERT(!m_xpcConnection);
 
 #if USE(EXTENSIONKIT)
-    auto handler = [](ThreadSafeWeakPtr<ProcessLauncher> weakProcessLauncher, _SEExtensionProcess* process, ASCIILiteral name, NSError* error)
+    auto handler = [](ThreadSafeWeakPtr<ProcessLauncher> weakProcessLauncher, ExtensionProcess& process, ASCIILiteral name, NSError *error)
```

Comment:
Given that you're doing a WTFMove() of the process in the implementation. I think we should be honest here and use `ExtensionProcess&&`. Otherwise it may be super confusing to the caller to get its variable moved out.